VersionControlUI/src/items/VDiffComparisonPair.cpp

```diff
@@ -112,16 +252,42 @@ void VDiffComparisonPair::scaleVisualizations()
 	if (componentType_)
 		componentType_->setScale(scale);
 
+	if (singleObjectPath_)
+		singleObjectPath_->setScale(scale);
+
 	if (oldVersionObjectPath_)
 		oldVersionObjectPath_->setScale(scale);
 
 	if (newVersionObjectPath_)
 		newVersionObjectPath_->setScale(scale);
+
+	if (nodeNotFoundIcon_)
+		nodeNotFoundIcon_->setScale(scale);
 }
 
 void VDiffComparisonPair::determineChildren()
 {
 	Super::determineChildren();
 	scaleVisualizations();
 }
+
+int VDiffComparisonPair::determineForm()
+{
+	static const int FORM_WITH_SINGLE_OBJECT_PATH = 0;
+	static const int FORM_CONTAINING_ONLY_NEW_NODE = 1;
+	static const int FORM_CONTAINING_ONLY_OLD_NODE = 2;
+	static const int FORM_WITH_TWO_OBJECT_PATHS = 3;
+
+	DiffComparisonPair* associatedNode = node();
```

Comment:
you don't need this. just use `node()` in all places below. This is an inline function so it will be plenty fast.